CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+Version 19.03
+-------------
+19.03.1
+  - Fixes:
+    - #1529 Update project PyPI credentials
+19.03.0
```

Comment:
Technically, this is not available on PyPi. So, we can merge both the change logs under `19.03.1` maybe?

@ahopkins @yunstanford @sjsadowski @vltr

Reply:
Agreed. If we really want to have it there for consistency, I would say to put everything under `19.3.1`, and then have a note for `19.3.0` that says something like "skipped for package management purposes, not released on PyPI".

Reply:
I hate that `pypi` changes the version to number based and removes leading zeroes. 🙄

Reply:
@seemethere There's be an open issue on the `setuptools` for forever on this https://github.com/pypa/setuptools/issues/308. 

Technically on the pypi side, `19.03.1 == 19.3.1` because of PEP-440

> All release segments involved in the comparison MUST be converted to a consistent length by padding shorter segments with zeros as needed.

Reply:
I know, but visually for calver `19.3.1` looks so much worse than `19.03.1` since it becomes inconsistent if you ever decide to release between October through December.

🤷‍♂️, I guess `setuptools` is the ultimate authority on versioning.